aiosparql/client.py

```diff
@@ -78,17 +79,18 @@ def format_field(self, value, format_spec):
         return indent(format(value, format_spec), self.indent)
 
 
-class SPARQLClient(aiohttp.ClientSession):
+class SPARQLClient:
     def __init__(self,
                  endpoint: Optional[str] = None,
                  update_endpoint: Optional[str] = None,
                  prefixes: Optional[List[Namespace]] = None,
                  graph: Optional[str] = None,
                  **kwargs):
-        super(SPARQLClient, self).__init__(**kwargs)
+        self._closed = False
         self.endpoint = endpoint or ENV['MU_SPARQL_ENDPOINT']
         self.graph = graph or IRI(ENV['MU_APPLICATION_GRAPH'])
         self.update_endpoint = update_endpoint or self.endpoint
+        self.session = aiohttp.ClientSession(**kwargs)
```

Comment:
Is the attribute (along with `graph`, `endpoint` etc) writable by design?
If not -- I suggest to convert them into python readonly properties by other PR.